homeassistant/components/switch/tellstick.py

```diff
@@ -31,16 +30,23 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
             "Failed to import tellcore")
         return
 
+    # pylint: disable=no-member
+    if telldus.TelldusCore.callback_dispatcher is None:
```

Comment:
I am slightly confused by this logic. If you're creating a dispatcher and a core that is connected to it, it will be accessible by the other tellstick component? And so if the other component gets loaded, the callback_dispatcher is no longer `None` and thus `telldus.TelldusCore()` is enough to create one attached to the dispatcher?